isolate.sh

```diff
@@ -1080,7 +1108,7 @@ function prev_fail_check() {
         check_CSCR "$MASTER_NS"
         #wait for cert manager to come back ready after scaling up
         local ns_list=$(gather_csmaps_ns)
-        wait_for_certmanager "${ns_list}"
+        wait_for_certmanager
```

Comment:
Hi @qpdpQ 

Is this redundant given we will have dedicated cert manager check after the `prev_fail_check`
<img width="1161" alt="Screenshot 2024-11-20 at 12 00 04 PM" src="https://github.com/user-attachments/assets/eda626e2-3bb3-4b52-a109-39f15acf9f61">